### PR TITLE
[IMP] various: improve english source terms

### PR DESCRIPTION
**Pull request by Yenthe666**
Description of the issue/feature this PR addresses: Improve the source terms

Current behavior before PR: Not the best terms

Desired behavior after PR is merged: Better terms

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
Thanks !
I guess we could fix a few more while we are at it.
```
$ grep -r "here is no" **.pot
addons/account/i18n/account.pot:"There is no Transfer Account defined in the accounting settings. Please "
addons/account/i18n/account.pot:msgid "There is no journal items in draft state to post."
addons/account/i18n/account.pot:"There is no need for regex delimiter, only the regex is needed. For instance if you want to extract the amount from\n"
addons/account/i18n/account.pot:"There is no tax cash basis journal defined for this company: \"%s\" \n"
addons/account/i18n/account.pot:msgid "There is nothing to reconcile."
addons/base_iban/i18n/base_iban.pot:msgid "There is no IBAN code."
addons/crm_iap_lead/i18n/crm_iap_lead.pot:"    <p>There is no more credits on your IAP Lead Generation account.<br/>\n"
addons/crm_iap_lead_website/i18n/crm_reveal.pot:"                    <p>There is no more credits on your IAP Lead Generation account.<br/>\n"
addons/delivery/i18n/delivery.pot:msgid "There is no matching delivery rule."
addons/gamification/i18n/gamification.pot:"There is no goal associated to this challenge matching your search.\n"
addons/google_drive/i18n/google_drive.pot:"There is no refresh code set for Google Drive. You can set it up from the "
addons/hr_contract/i18n/hr_contract.pot:"title=\"There is something wrong with the contract. Either there is no "
addons/hr_contract/i18n/hr_contract.pot:"there is no running contract for this employee or employee's contract is "
addons/hr_presence/i18n/hr_presence.pot:"                    Exception made if there was a mistake of ours, it seems that you are not at your office and there is not request of leaves from you.<br/>\n"
addons/hr_presence/i18n/hr_presence.pot:"Exception made if there was a mistake of ours, it seems that you are not at your office and there is not request of leaves from you.\n"
addons/hr_presence/i18n/hr_presence.pot:msgid "There is no professional email address for this employee."
addons/hr_presence/i18n/hr_presence.pot:msgid "There is no professional mobile for this employee."
addons/im_livechat/i18n/im_livechat.pot:msgid "There is no rating for this channel at the moment"
addons/lunch/i18n/lunch.pot:"There is no previous order recorded. Click on \"My Lunch\" and then create a"
addons/lunch/i18n/lunch.pot:msgid "There is no product available today"
addons/mass_mailing/i18n/mass_mailing.pot:msgid "There is no recipients selected."
addons/mass_mailing_sms/i18n/mass_mailing_sms.pot:msgid "There is no recipients selected."
addons/mrp/i18n/mrp.pot:"There is no Bill of Material of type manufacture or kit found for the "
addons/point_of_sale/i18n/point_of_sale.pot:"There is no Chart of Accounts configured on the company. Please go to the "
addons/point_of_sale/i18n/point_of_sale.pot:"There is no cash payment method available in this point of sale to handle the change.\n"
addons/point_of_sale/i18n/point_of_sale.pot:msgid "There is no cash register for this PoS Session"
addons/pos_reprint/i18n/pos_reprint.pot:msgid "There is no previous receipt to print."
addons/pos_restaurant/i18n/pos_restaurant.pot:msgid "There is no order lines"
addons/project/i18n/project.pot:msgid "There is no rating for this object at the moment"
addons/purchase_requisition/i18n/purchase_requisition.pot:msgid "You cannot confirm agreement '%s' because there is no product line."
addons/purchase_stock/i18n/purchase_stock.pot:"There is no matching vendor price to generate the purchase order for product"
addons/rating/i18n/rating.pot:msgid "There is no rating for this object at the moment."
addons/sale/i18n/sale.pot:"There is no invoiceable line. If a product has a Delivered quantities "
addons/sale_purchase/i18n/sale_purchase.pot:"There is no vendor associated to the product %s. Please define a vendor for "
addons/sale_timesheet/i18n/sale_timesheet.pot:msgid "There is no timesheet for now."
addons/sale_timesheet/i18n/sale_timesheet.pot:"This timesheet line cannot be billed: there is no Sale Order Item defined on"
addons/stock/i18n/stock.pot:"Take From Stock, if Unavailable, Trigger Another Rule: the products will be taken from the available stock of the source location.If there is no stock available, the system will try to find a  rule to bring the products in the source location."
addons/stock_account/i18n/stock_account.pot:"There is no valuation layers. Valuation layers are created when some product"
addons/web/i18n/web.pot:msgid "There is no available image to be set as cover."
addons/web/i18n/web.pot:"Try to add some records, or make sure that there is no\n"
addons/website/i18n/website.pot:msgid "There is no data currently available."
addons/website/i18n/website.pot:msgid "There is no email linked this visitor."
addons/website/i18n/website.pot:"There is no website available for this company. You could create a new one."
addons/website_forum/i18n/website_forum.pot:"there is no actual problem to be solved: “I’m curious if other people feel "
addons/website_links/i18n/website_links.pot:msgid "There is no data to show"
addons/website_livechat/i18n/website_livechat.pot:msgid "There is no public livechat channel to show."
addons/website_livechat/i18n/website_livechat.pot:msgid "There is no rating for this channel for now."
addons/website_sale/i18n/website_sale.pot:msgid "There is no confirmed order from the website"
addons/website_sale/i18n/website_sale.pot:msgid "There is no recent confirmed order."
addons/website_sale/i18n/website_sale.pot:msgid "There is no unpaid order from the website yet"
addons/website_slides/i18n/website_slides.pot:msgid "There is no quiz"
addons/website_slides/i18n/website_slides.pot:msgid "There is no rating for these courses at the moment"
addons/website_slides_forum/i18n/website_slides_forum.pot:"there is no actual problem to be solved: “I’m curious if other people feel "
addons/website_slides_survey/i18n/website_slides_survey.pot:msgid "The related course slide when there is no membership information"
odoo/addons/base/i18n/base.pot:"and wishes. There is no central tag repository across clients."
```
Even if some of them are correct.

cf https://english.stackexchange.com/a/26634

> There are no comments.

But

> There is no newspaper in my driveway today.

So

> There is no tax cash basis journal defined for this company

is correct. But

> There is no public livechat channel to show.

is not

**Comment by Yenthe666** [comment]
@mart-e done, I think I've got them all now :)

File: addons/website_livechat/views/website_livechat.xml [hunk 1]
```diff
@@ -208,7 +208,7 @@
                         <div class="row mt32 mb32">
                             <t t-if="not len(channels)">
                                 <div class="col-lg-6 offset-lg-3">
-                                    There is no public livechat channel to show.
+                                    There is are public livechat channels to show.
```
Comment (mart-e) [inline addons/website_livechat/views/website_livechat.xml]: the "no" was lost here

**Comment by mart-e** [comment]
Impressive, thanks !

@robodoo r+

**Comment by Yenthe666** [comment]
Thanks a lot :)

**Pull request closed without merging**